apps/buildog/src/app/organizations/[organizationId]/web/page.tsx

```diff
@@ -48,26 +39,74 @@ export default function DropdownMenuCheckboxes() {
               onCheckedChange={() => setHeader("DefaultHeader")}
               className="cursor-pointer"
             >
-              Do you Prefer Simplicity?
+              Default Header
```

Comment:
I think these should be Card right now Header
```suggestion
              Default Card
```